content/syllabuses/java-part-1.md

```diff
@@ -5,17 +5,17 @@ _db_id: 7
 
 - {{% contentlink path="agile/agile-scrum" %}}
 - {{% contentlink path="topics/solo-learn/java/basic-concepts" %}}
-- {{% contentlink path="workshops/big-picture-web-dev" %}}
+- {{% contentlink path="topics/big-picture-web-dev" %}}
 - {{% contentlink path="topics/solo-learn/java/conditionals-and-loops" %}}
 - {{% contentlink path="topics/solo-learn/java/arrays" %}}
+- {{% contentlink path="topics/clean-code/java" %}}
 - {{% contentlink path="topics/clean-code" %}}
```

Comment:
can probably remove - {{% contentlink path="topics/clean-code" %}}
since - {{% contentlink path="topics/clean-code/java" %}} is here